tt_metal/hw/inc/blackhole/dev_mem_map.h

```diff
@@ -52,7 +52,7 @@
 #define MEM_BOOT_CODE_BASE 0
 #define MEM_L1_BARRIER 12
 #define MEM_MAILBOX_BASE 16
-#define MEM_MAILBOX_END (MEM_MAILBOX_BASE + 1344)
+#define MEM_MAILBOX_END (MEM_MAILBOX_BASE + 1348)
```

Comment:
how do we calculate this value? Could you add comment or make it depend on sum of other defines directly?